src/components/pipelines-overview/PipelineRunsTotalCard.tsx

```diff
@@ -43,51 +43,44 @@ const PipelinesRunsTotalCard: React.FC<PipelinesRunsDurationProps> = ({
   }
 
   const date = getDropDownDate(timespan).toISOString();
-  const filter = `data.metadata.labels.contains("pipelinesascode.tekton.dev/repository")&&data.status.startTime>timestamp("${date}")`;
 
-  React.useEffect(() => {
+  const getSummaryData = (filter, setData) => {
     getResultsSummary(namespace, {
       summary: 'total',
       data_type: DataType.PipelineRun,
       filter,
     })
       .then((response) => {
-        setRepoRun(response.summary[0].total);
+        setData(response.summary[0].total);
       })
       .catch((e) => {
-        console.error('Error in getSummary', e);
+        console.error('Unable to Post', e);
```

Comment:
Nit: revert to Error in getSummary

Reply:
ya had issue selecting 1 or the other, will revert to error in get summary.